### ISSUE TITLE
Can we use the worker script to add basic footer elements?

### ISSUE THREAD

**Issue by danblah** [ResetNetwork/recoding#11]

**Comment by rudietuesdays** [comment]
Yup, added some text to https://recode2.reset.tech/ to show an example

**Comment by danblah** [comment]
Awesome... can we add links for all the More Info pages and a hello@recoding.tech?

link1 | Link2 | link3 | hello@recoding.tech

or some such?

**Comment by rudietuesdays** [comment]
Yup! Example set here: https://recoding.tech/

Let me know how it looks.

**Comment by danblah** [comment]
Awesome. Can we add Home and a light grey background shade?
